src/components/ResourceList/tests/ResourceList.test.tsx

```diff
@@ -750,6 +753,118 @@ describe('<ResourceList />', () => {
       resourceList.update();
       expect(resourceList.find(BulkActions).prop('selectMode')).toBe(false);
     });
+
+    describe('focus', () => {
+      describe('large screen', () => {
+        it('focuses the checkbox in the bulk action when the plain CheckableButton is clicked', () => {
+          const resourceList = mountWithAppProvider(
+            <ResourceList
+              items={itemsWithID}
+              renderItem={renderItem}
+              promotedBulkActions={promotedBulkActions}
+            />,
+          );
+
+          const selectAllCheckableButton = resourceList.findWhere(
+            (wrap) => wrap.is(CheckableButton) && wrap.prop('plain') === true,
+          );
+
+          trigger(selectAllCheckableButton, 'onToggleAll');
+
+          const deselectAllCheckbox = resourceList
+            .findWhere(
+              (wrap) => wrap.is(CheckableButton) && !wrap.prop('plain'),
+            )
+            .find('input[type="checkbox"]');
```

Comment:
Just curious why you didn't include this in `findWhere` or before the `findWhere` (both should reduce the number of operations, although probably by a negligible amount)(not asking for a code change, just curious)

Reply:
🤔 the findWhere will return the `wrap` based on the filtering condition. I'm after the checkbox inside the wrap. Putting the `find` before would not allow me to `findWhere` based on parents asaik. In any case, this has been simplified, because as you suggested I made the findWhere into a function.